modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -118,6 +130,24 @@ class HttpSigningConfiguration(object):
         self.private_key_path = private_key_path
         self.private_key_passphrase = private_key_passphrase
         self.signing_algorithm = signing_algorithm
+        self.hash_algorithm = hash_algorithm
+        if signing_scheme == SCHEME_RSA_SHA256:
+            if self.hash_algorithm == None:
```

Comment:
The standard python practice is to use `is None` when comparing to None.
Can you change this and the below lines to that pattern?
- 141
- 147
If a user passes in an instance of a custom class then using `self.hash_algorithm == None` could be True when we don't expect it to be per https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or